net/private.go

```diff
@@ -130,6 +135,13 @@ func IsPublicAddr(a ma.Multiaddr) bool {
 	return isPublic
 }
 
+// isSubDomain checks if child is sub domain of parent. It also returns true if child and parent are
+// the same domain.
+// Parent must have a "." prefix.
+func isSubDomain(child, parent string) bool {
```

Comment:
Nit: I believe the more correct capitalization would be `isSubdomain`.

Reply:
indeed.